src/v2/Apps/Home/Components/HomeArtworkModules.tsx

```diff
@@ -10,6 +10,8 @@ import { MyBidsQueryRenderer } from "v2/Apps/Auctions/Components/MyBids/MyBids"
 import { HomeFeaturedShowsLazyQueryRenderer } from "./HomeFeaturedShows"
 import { compact } from "lodash"
 
+const EXCLUDED_MODULES = ["popular_artists", "generic_gene"]
```

Comment:
@dzucconi - It seems like this should be elevated to MP. Thoughts?

Reply:
[Already is!](https://github.com/artsy/force/pull/8390#discussion_r706443835)

Reply:
Ah, looks like I was on the wrong field: 

<img width="317" alt="Screen Shot 2021-09-10 at 1 13 06 PM" src="https://user-images.githubusercontent.com/236943/132912219-5a0299a6-5246-4d1b-9832-4f1f6860bfe4.png">

Will fix 👍

Reply:
(Ah -- was looking at v1.)

Reply:
Ugh, this happens to me frequently